src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCaseBehavior.cs

```diff
@@ -8,16 +8,16 @@ namespace System.Text.RegularExpressions
     /// <summary>
     /// When a regular expression specifies the option <see cref="RegexOptions.IgnoreCase"/> then comparisons between the input and the
     /// pattern will made case-insensitively. In order to support this, we need to define which case mappings shall be  used for the comparisons.
-    /// A case mapping exists whenever you have two characters 'A' and 'B', where either 'A' is the ToLower() representation of 'B'. Note that we don't
-    /// consider a mapping when the only relationship between 'A' and 'B' is that one is the ToUpper() representation of the other. This is for backwards compatibility
-    /// since, in Regex, we have only consider ToLower() for case insensitive comparisons. Given the case mappings vary depending on the culture, Regex supports 3 main
-    /// different behaviors or mappings: Invariant, NonTurkish, and Turkish. This is in order to match the behavior of all .NET supported cultures current behavior for
-    /// ToLower().
+    /// A case mapping exists whenever you have two characters 'A' and 'B', where either 'A' is the ToLower() representation of 'B' or both 'A' and 'B' lowercase to the
```

Comment:
It might be worth explicitly mentioning that there are no cases where a.ToLower() == b but a.ToLower() != b.ToLower() (i.e. b != b.ToLower()).  If there were such a case, these rules would then lead to an inconsistency with how we handle backreferences, which only considers a.ToLower() == b.ToLower().

Reply:
Should we assert this in a tes that loops through all the characters over all the cultures? Culture stuff changes periodically and this code will be around for years.